common/src/main/java/io/netty/util/internal/Cleaner0.java

```diff
@@ -29,10 +29,9 @@
  *
  * For more details see <a href="https://github.com/netty/netty/issues/2604">#2604</a>.
  */
-final class Cleaner0 {
+final class Cleaner0 implements Cleaner {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Initialization circularity between io.netty.util.internal.Cleaner0 and io.netty.util.internal.CleanerJava9 [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=findbugs%3AIC_INIT_CIRCULARITY)
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Initialization circularity between io.netty.util.internal.Cleaner0 and io.netty.util.internal.PlatformDependent [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=findbugs%3AIC_INIT_CIRCULARITY)
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Initialization circularity between io.netty.util.internal.Cleaner0 and io.netty.util.internal.PlatformDependent0 [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=findbugs%3AIC_INIT_CIRCULARITY)